### PR TITLE
v5.37 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by itao — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]

**Review by chenilim — APPROVED** [review]
Focalboard section LGTM. Thanks!

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+## Release v5.37 - [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html)
+
+**Release Day: 2021-07-16**
+
+### Deprecations
+ - The ``platform`` binary and “--platform” flag have been deprecated. If you are using the “--platform” flag or are using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the mattermost binary.
+
+### Important Upgrade Notes
+ - [Collapsed Reply Threads](https://mattermost.com/blog/collapsed-reply-threads-beta/) are available as beta in Mattermost Server v5.37 and later. It’s expected that you may experience bugs as we stabilize the feature. In particular, please be aware of [the known issues documented here](https://docs.mattermost.com/help/messaging/organizing-conversations.html#known-issues).
+ - v5.37 adds support for emoji standard v13.0. If you have added a custom emoji in the past that uses one of the new system names, then it is going to get overwritten by the system emoji. The workaround is to change the custom emoji name.
```
Comment (cwarnermm) [inline source/administration/changelog.md:16]:
```suggestion
 - v5.37 adds support for emoji standard v13.0. If you have added a custom emoji in the past that uses one of the new system names, then that custom emoji is going to get overwritten by the system emoji. The workaround is to change the custom emoji name.
```

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+#### English-Australian Language Support
+ - Mattermost is now available in English-Australian.
+
+### Improvements
+
+#### User Interface (UI)
+ - In the at-mention autocomplete, the user’s nickname is no longer shown when (you) is present.
+ - Updated the help text on the **Add Users** channel modal.
+ - Added the ability to upload ``.jpeg`` files on Linux. Uploading ``.jpg`` files was already supported.
+ - Before anything is typed, a list of channels you are a member of is now shown and sorted by recency in the channel switcher. The number of channels shown in the channel switcher in the default state are capped to maximum of 20. If there is no recent channel activity (in the case of new users), an alphabetical list of channels is shown displaying **My Channels** first and other **Public Channels** next, capped at a maximum of 20 by default.
```
Comment (cwarnermm) [inline source/administration/changelog.md:47]:
```suggestion
 - Before anything is typed, a list of channels you are a member of is now shown and sorted by recency in the channel switcher. The number of channels shown in the channel switcher in the default state are capped to a maximum of 20. If there is no recent channel activity (in the case of new users), an alphabetical list of channels is shown displaying **My Channels** first and other **Public Channels** next, capped at a maximum of 20 by default.
```

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Polish, German, and Italian languages were downgraded to beta as they are [no longer actively maintained](https://handbook.mattermost.com/contributors/contributors/localization#translation-quality).
+ - Added the ability in the custom status tooltips, status dropdown and profile popover to select an expiry time for custom statuses. Also added new components ``ExpiryMenu`` and ``DateTimeInput`` to the custom status modal.
```
Comment (cwarnermm) [inline source/administration/changelog.md:49]:
```suggestion
 - Added the ability in the custom status tooltips, status dropdown, and profile popover to select an expiry time for custom statuses. Also added new components ``ExpiryMenu`` and ``DateTimeInput`` to the custom status modal.
```

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+#### Performance
+ - Added some improvements to typing performance.
+
+#### Administration
+ - Improved memory performance for large image uploads, particularly PNGs with transparency.
+ - Optimized the bulk import process by no longer requiring the server to write the incoming archive to the filesystem when unzipping it.
+ - Added channel restore and channel privacy change endpoints to the local mode using the System bot.
+
+### Bug Fixes
+ - Fixed an issue where users were unable to set a custom status emoji via slash command by adding the logic for detecting unicode emoji and setting it as custom status emoji via slash commands.
```
Comment (cwarnermm) [inline source/administration/changelog.md:60]:
```suggestion
 - Fixed an issue where users were unable to set a custom status emoji via slash command. Added the logic for detecting unicode emoji and setting it as a custom status emoji via slash commands.
```

**Review by cwarnermm — COMMENTED** [review]
Suggestions provided inline

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+#### English-Australian Language Support
+ - Mattermost is now available in English-Australian.
+
+### Improvements
+
+#### User Interface (UI)
+ - In the at-mention autocomplete, the user’s nickname is no longer shown when (you) is present.
+ - Updated the help text on the **Add Users** channel modal.
+ - Added the ability to upload ``.jpeg`` files on Linux. Uploading ``.jpg`` files was already supported.
+ - Before anything is typed, a list of channels you are a member of is now shown and sorted by recency in the channel switcher. The number of channels shown in the channel switcher in the default state are capped to a maximum of 20. If there is no recent channel activity (in the case of new users), an alphabetical list of channels is shown displaying **My Channels** first and other **Public Channels** next, capped at a maximum of 20 by default.
```
Comment (esethna) [inline source/administration/changelog.md:47]: This feels a bit long winded, Propose: "The Channel Switcher now displays recently viewed channels when launched"

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Polish, German, and Italian languages were downgraded to beta as they are [no longer actively maintained](https://handbook.mattermost.com/contributors/contributors/localization#translation-quality).
+ - Added the ability in the custom status tooltips, status dropdown, and profile popover to select an expiry time for custom statuses. Also added new components ``ExpiryMenu`` and ``DateTimeInput`` to the custom status modal.
```
Comment (esethna) [inline source/administration/changelog.md:49]: Focus on end user benefit: "Custom statuses can now be set to expire after common time intervals or custom selected dates and times.

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Added two new fields in ``CustomStatus`` struct and modified the APIs to validate and handle them.
+
+### Go Version
+ - v5.37 is built with Go ``1.15.5``.
+
+### Open Source Components
+ - Removed ``reselect`` from https://github.com/mattermost/mattermost-webapp/.
+
+### Known Issues
+ - ``config.json`` can reset when running the command ``systemctl restart mattermost``, and when running any commands that write to the config (e.g. ``config`` or ``plugin``) [MM-33752](https://mattermost.atlassian.net/browse/MM-33752), [MM-32390](https://mattermost.atlassian.net/browse/MM-32390).
```
Comment (esethna) [inline source/administration/changelog.md:94]: Let's please add a bullet here at the top linking to the CRT doc known issues section

**Review by esethna — CHANGES_REQUESTED** [review]
Added a few comments

**Review by esethna — APPROVED** [review]

**Pull request merged**